src/main/java/internet/shop/db/Storage.java

```diff
@@ -1,19 +1,43 @@
 package internet.shop.db;
 
+import internet.shop.model.Car;
+import internet.shop.model.Driver;
 import internet.shop.model.Manufacturer;
 import java.util.ArrayList;
 import java.util.List;
 
 public class Storage {
     private static long manufacturerId = 1;
+    private static long carId = 1;
+    private static long driverId = 1;
     private static final List<Manufacturer> dbManufacturer = new ArrayList<>();
```

Comment:
```suggestion
    private static final List<Manufacturer> manufacturers = new ArrayList<>();
```
the same with others

Reply:
done